frontend-react/package.json

```diff
@@ -65,7 +68,7 @@
         "e2e": "cypress run",
         "e2e:open": "cypress open",
         "test": "react-scripts test",
-        "test:ci": "CI=true yarn --verbose test --coverage --verbose",
+        "test:ci": "CI=true yarn --verbose test --coverage --verbose --silent",
```

Comment:
you know what Tom, they probably do

it feels weird but I still like this setup because it a) gives us clean output on most test runs, and b) allows us to get a full dirty output for debugging by removing the `--silent` flag

the original intent with the silent flag was to get rid of logs that appear on successful tests for things like error handling and bad requests without having to dig into the code on each test, or globally within a jest config, to overwrite console behavior.